Source/WebCore/dom/make_names.pl

```diff
@@ -759,145 +833,562 @@ END
     close F;
 }
 
-sub findMaxTagLength
+sub byTagNameOrder
+{
+    tagCount($allTags{$a}{localName}) <=> tagCount($allTags{$b}{localName})
+        || $allTags{$a}{elementEnumValue} cmp $allTags{$b}{elementEnumValue}
+}
+
+sub printTagNameHeaderFile
+{
+    my ($headerPath) = shift;
+    my $F;
+    open F, ">$headerPath";
+
+    printLicenseHeader($F);
+    print F "#pragma once\n";
+    print F "\n";
+    print F "#include <wtf/EnumeratedArray.h>\n";
+    print F "#include <wtf/NeverDestroyed.h>\n";
+    print F "#include <wtf/text/AtomString.h>\n";
+    print F "\n";
+    print F "// Windows headers define this macro, which conflicts with an HTML tag name.\n";
+    print F "#if defined(small)\n";
+    print F "#undef small\n";
```

Comment:
Why not do the same thing as other keywords and just append _ instead?
undef'ing windows macro sounds like a trouble down the line
when someone inevitably include these headers in Windows port.

Reply:
Sure, can do that.